app/classes/api2/herbarium_record_api.rb

```diff
@@ -40,11 +40,11 @@ def query_params
         herbaria: parse_array(:herbarium, :herbarium, as: :id),
         observations: parse_array(:observation, :observation, as: :id),
         has_notes: parse(:boolean, :has_notes),
-        initial_dets: parse(:string, :initial_det, help: 1),
-        accession_numbers: parse(:string, :accession_number, help: 1),
+        initial_det: parse(:string, :initial_det, help: 1),
+        accession: parse(:string, :accession_number, help: 1),
```

Comment:
Is this changing the parameters for the API?  Looking over the last couple weeks of logs, I do see some references to these these fields.  I'm not sure yet if they are due to API calls, but in general changing parameters without providing backwards compatibility or changing the version number of the API is a bad practice.

Reply:
Did a bit more testing and I think the API parameters for GET and POST are not changed.  See test/classes/api2/herbarium_records_test.rb.  However, it does seem weird that the GET returns:
            "initial_determination": "Tricholoma arvernense",
but the POST seems to want:
      initial_det: @initial_det,

However, that's out of scope for this PR.